### DISCUSSION TITLE
Remove redundant files & folders in the root folder

### DISCUSSION THREAD

**Discussion by toby-thanathip** [nimblehq/android-templates#432] [RFCs]
Related to: https://github.com/nimblehq/android-templates/pull/347#discussion_r1022586445

We don't need `gradle` & `gradle-wrapper.properties` at the root level of `android-templates`, so let's remove them.

![image](https://user-images.githubusercontent.com/18277915/223603767-8a871eca-4963-4496-a3ac-635442435326.png)

**Comment by chornerman** [comment]
Voted 🚀
